babble-backend/database/db.js

```diff
@@ -110,4 +110,7 @@ module.exports = {
     writeUsers,
     writePosts,
     writeAuth
-}
\ No newline at end of file
+}
+
+/* Load data whenever database is required */
+loadData();
```

Comment:
Don't think this is necessary since we only need to load data from the json files once (when the server is being started). Moving it back to server.js is probably the move.

Reply:
Yeah this makes more sense, I forgot that the data gets updated in db and the .json files regardless of what function in our code is calling the update functions.